web_external/Featuresets/FeaturesetsView.js

```diff
@@ -24,6 +24,11 @@ isic.views.FeaturesetsView = isic.View.extend({
         }, this);
         this.featuresets.fetch();
 
+        // TODO: Use the more general 'update' event, once Backbone is upgraded from Girder
+        this.listenTo(this.featuresets, 'remove', function () {
+            this.render();
+        }, this);
```

Comment:
`listenTo` uses the object that it's called on as the context, so the `this` argument shouldn't be here. It's an error that it's written like that above, so that argument can be removed as well.